libs/docs/src/docs/no-version/what-is-a-devfile.md

```diff
@@ -3,16 +3,14 @@ title: What is a devfile
 description: What is a devfile
 ---
 
-You can use devfiles to automate and simplify your development process
-by adopting the existing devfiles that are available in the [public community registry](https://registry.devfile.io/viewer)
-or by authoring your own devfiles to record custom instructions to 
-configure and run your build environment as a YAML-formatted text file.
-You can make these devfiles available in the supporting build tools and 
-IDEs that can automatically process the devfile instructions to configure 
-and build a running application from a development project.
-
-Using the recommended best practices from the devfile, the tools and IDE
-can:
+`devfile.yaml` describes how to configure and run build environment for
+your project in a development container. You can create devfile from scratch or
+adopt existing from [public community registry](https://registry.devfile.io/viewer).
+
+Certain build tools and IDEs can automatically process the devfile
+to configure and build a running application from a development project.
```

Comment:
```suggestion
to configure, build, and deploy an application from a development project.
```

Reply:
I am pretty sure application deployment is out of devfile responsibility. For proper deployment, one need stuff like Ansible and OpenTofu.

Reply:
> I am pretty sure application deployment is out of devfile responsibility. For proper deployment, one need stuff like Ansible and OpenTofu.

@abitrolly  Not entirely true, there might be a misunderstanding here between the deployment environments, i.e. staging, production, and what I mean which is a development live deployment a developer uses to see the applied changes.

For some consumers there are features perform a simple application deploy, e.g. when you run `odo deploy`, though most production system do use technologies as the ones you are mentioning.

Reply:
If it is impossible to choose where and how to deploy, then it is not deployment. Running application in containers with `docker-compose` is not deployment. Using Ansible to setup `docker-compose` on remote server and update application there is deployment.

Reply:
Applied. Although if the "devfile" role is to help build and reload application automatically while editing, then the former text better reflects it.